components/ws-daemon/pkg/controller/workspace_controller.go

```diff
@@ -125,34 +126,53 @@ func (wsc *WorkspaceController) Reconcile(ctx context.Context, req ctrl.Request)
 	glog.WithField("workspaceID", workspace.Name).WithField("phase", workspace.Status.Phase).Debug("Reconcile workspace")
 
 	if workspace.Status.Phase == workspacev1.WorkspacePhaseCreating ||
-		workspace.Status.Phase == workspacev1.WorkspacePhaseInitializing ||
-		workspace.Status.Phase == workspacev1.WorkspacePhaseRunning {
```

Comment:
what's the reason for not checking workspaceInit on the running state? Wouldn't it be possible for daemon to miss a workspace's events up until the Running state if daemon e.g. restarts?